tests/config/pytest_modules.yml

```diff
@@ -1,12 +1,6 @@
-preproc/gibbs:
-  - modules/nf-scil/preproc/gibbs/**
-  - tests/modules/nf-scil/preproc/gibbs/**
-preproc/n4:
-  - modules/nf-scil/preproc/n4/**
-  - tests/modules/nf-scil/preproc/n4/**
-preproc/normalize:
-  - modules/nf-scil/preproc/normalize/**
-  - tests/modules/nf-scil/preproc/normalize/**
+image/resample:
+  - modules/nf-scil/image/resample/**
+  - tests/modules/nf-scil/image/resample/**
```

Comment:
Module image/resample has already been converted to nf-test. It shouldn't be here (probably a merge conflict error).